benchmark/rgw_benchmark_setup.yml

```diff
@@ -0,0 +1,29 @@
+---
+- hosts:
+    - mons
```

Comment:
Fixed

Reply:
We will need to run this on the mons because it errors out on the rgws containers
radosgw-admin user create --uid='rgwbench' --display-name='Rgw Benchmark'
2018-02-15 22:04:52.672883 7f20bb3d3cc0 -1 auth: unable to find a keyring on /etc/ceph/ceph.client.admin.keyring,/etc/ceph/ceph.keyring,/etc/ceph/keyring,/etc/ceph/keyring.bin,: (2) No such file or directory
2018-02-15 22:04:52.674736 7f20bb3d3cc0 -1 monclient: authenticate NOTE: no keyring found; disabled cephx authentication
2018-02-15 22:04:52.674744 7f20bb3d3cc0  0 librados: client.admin authentication error (95) Operation not supported

Reply:
:+1: ok that works then. lets not add extra perms to the rgws that seems worse